### DISCUSSION TITLE
Counter Strike Source tick rate 100 problem

### DISCUSSION THREAD

**Discussion by menthos13** [GameServerManagers/LinuxGSM#4513] [Q&A]
Hi guys, how I can enable tick rate 100 on my CSS server? I did install Tickrate_Enabler.so
en put this on my server.cfg :

sv_tickrate 100
sv_maxupdaterate 101
sv_maxcmdrate 101
sv_minupdaterate 100
sv_mincmdrate 100
sv_minrate 20000
sv_maxrate 30000

Now it's stuck on 66 any idea?

Regards

**Comment by iErcann** [comment]
you need a tickrate enabler
seems like this is the latest working version: https://github.com/idk1703/TickrateEnabler/releases/tag/v0.5-upd

https://github.com/daemon32/tickrate_enabler/issues/5

after that setting -tickrate 100 worked for me

https://docs.linuxgsm.com/configuration/start-parameters#basic-example
```startparameters="-game cstrike -strictportbind -ip ${ip} -port ${port} +clientport ${clientport} +tv_port ${sourcetvport} +sv_setsteamaccount +map ${defaultmap} +servercfgfile ${servercfg} -maxplayers ${maxplayers} -tickrate 100"```
